packages/account/src/Components/poi/idv-status/idv-failed/idv-failed.tsx

```diff
@@ -319,15 +320,15 @@ const IdvFailed = ({
                     })}
                 >
                     <FormBody className='form-body' scroll_offset={setScrollOffset()}>
-                        <Text size={is_mobile ? 'xs' : 's'} weight='bold' align='center'>
+                        <Text size={isMobile ? 'xs' : 's'} weight='bold' align='center'>
                             <Localize i18n_default_text='Your identity verification failed because:' />
                         </Text>
                         {(status?.error_msg || idv_failure?.failure_message) && (
                             <HintBox
                                 className={classNames('proof-of-identity__failed-message', 'hint-box-layout')}
                                 icon='IcAlertDanger'
                                 message={
-                                    <Text as='p' size={is_mobile ? 'xxs' : 'xs'} data-testid={mismatch_status}>
+                                    <Text as='p' size={isMobile ? 'xxs' : 'xs'} data-testid={mismatch_status}>
```

Comment:
here also

Reply:
Hey @utkarsha-deriv
As we don't have design for all screens and I checked on tablet view xs is too small for tablet view.